### PR TITLE
Add com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0

### PR DESCRIPTION
**Pull request by colmsnowplow**

### CODE REVIEW

**Comment by knservis** [comment]
@colmsnowplow Can we call this gdpr_context instead of just gdpr for uniformity

**Comment by colmsnowplow** [comment]
> @colmsnowplow Can we call this gdpr_context instead of just gdpr for uniformity

The standard is to leave it out. I can see there are some that have `_context` but those are the exception.

File: schemas/com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          "description": "GDPR basis for data collection & processing"
+        },
+        "documentId": {
+          "type": ["string", "null"],
+          "maxLength": 255,
+          "description": "ID for document detailing basis for processing"
+        },
+        "documentVersion": {
+          "type": ["string", "null"],
+          "maxLength": 5,
```
Comment (yalisassoon) [inline schemas/com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0]: Suggest making this longer e.g. 16 characters, so you can accommodate e.g. `12.4.2`

Comment (colmsnowplow) [reply schemas/com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0]: Yup should've thought of that, fixing.

**Comment by colmsnowplow** [comment]
> LGTM, except from the name gdpr could mean anything gdpr_context is much more informative. I woudl definitely change the name.

I also prefer appending `_context` to the names of contexts. But the already established convention is to leave it out, so it doesn't make sense to break it. Most of the contexts follow this convention, and indeed it's what we recommend our customers follow in onboarding.

@chuwy I've never released to Iglu Central before, is there some process beyond asking you/someone else to merge the PR?

**Comment by alexanderdean** [comment]
Looks fine to me. Does it not need JSON Paths and a Redshift table defn?

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,8 @@
+Release 105 (2019-06-26)
+-----------------------
+
```
Comment (chuwy) [inline CHANGELOG]: Missing newline.

File: schemas/com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0 [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "gdpr",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+
+    "type": "object",
+    "properties": {
+        "basisForProcessing": {
+          "enum": ["consent", "contract", "legal obligation", "vital interest", "public task", "legitimate interests"],
```
Comment (chuwy) [inline schemas/com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0]: Do we have a control over these values? I have a slight preference towards enums without spaces.

File: CHANGELOG (hunk 2 above)
Comment (colmsnowplow) [reply CHANGELOG]: Fixed.

**Comment by colmsnowplow** [comment]
Added JSONpath files, removed whitespace from enums and removed newline from CHANGELOG. 

Hoping this is GTG now but let me know if not @alexanderdean and @chuwy :)

**Comment by alexanderdean** [comment]
Let's take the non-essential reviewers off this. If everybody else ticks, I can release tonight 👍

**Comment by colmsnowplow** [comment]
Thanks Alex, all necessary approvals given now!

File: schemas/com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0 [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "gdpr",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+
+    "type": "object",
+    "properties": {
+        "basisForProcessing": {
+          "enum": ["consent", "contract", "legal_obligation", "vital_interest", "public_task", "legitimate_interests"],
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0]: Should be `vital_interests` note the trailing s for consistency with legitimate_interests. Confirmed on https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/lawful-basis-for-processing/

Comment (colmsnowplow) [reply schemas/com.snowplowanalytics.snowplow/gdpr/jsonschema/1-0-0]: Thanks for flagging @alexanderdean , fixed now. :)

**Pull request merged**